index.html

```diff
@@ -55,7 +56,7 @@
         <div class="row">
           <div class="col-md-12 text-center">
             <div class="mdi-material-light-blue">
-              <h1>A GUI for BOSH</h1>
+              <h1>The GUI for BOSH</h1>
```

Comment:
fixed in https://github.com/dozer-cloud/dozer_design/commit/49976dc5234792cd36bec09576747049f07c42d8